src/backend/index.js

```diff
@@ -10,7 +10,7 @@ const MIGRATIONS_FOLDER_PATH = new URL(
 ).pathname
 
 const DEFAULT_CONFIG_PATH = new URL(
-  './node_modules/@mapeo/default-config/dist/mapeo-default-config.comapeocat',
+  'node_modules/@comapeo/default-categories/dist/comapeo-default-categories.comapeocat',
```

Comment:
think this works fine but might be safer to explicitly write this as a relative path
```suggestion
  './node_modules/@comapeo/default-categories/dist/comapeo-default-categories.comapeocat',
```

Reply:
https://github.com/digidem/comapeo-mobile/pull/1563/commits/8b8e2d87f03795683b4ca68c9cb01c539d988d2e